### PR TITLE
testing writing to bot.

### PR DESCRIPTION
**Pull request by kearstenprince**
will you try running this with your bot and let me know the outcome?

### CODE REVIEW

File: antidox/wikiwatcher.py [hunk 1]
```diff
@@ -50,21 +51,37 @@ def log_event(apikey_data, toxicity, dlp, change):
...
+    content = (u'user:{user} namespace:{namespace} bot:{bot} comment:{comment}'+
+               'title:{title}'.format(**change)+"\n"+str(text)+"\n"
+               +"contains TOXICITY?:"+"Yes"+"\n"+
+               str(perspective_response['attributeScores']
+                   ['TOXICITY']['summaryScore']['value'])+"\n"
+               +"=========================================="+"\n")
+    message = "\n\n{}\n{} --~~~~".format(heading, content)
+    page.save(summary="Testing", watch=None, minor=False, botflag=True,
+              force=False, async=False, callback=None,
+              apply_cosmetic_changes=None, appendtext=message)
```
Comment (dborkan) [inline antidox/wikiwatcher.py]: There is a lot of duplicate code between this code for toxicity and the code further up for pii.  Can you put this in a function that removes the duplicated code?

File: antidox/wikiwatcher.py [hunk 2]
```diff
@@ -35,9 +36,7 @@ def log_event(apikey_data, toxicity, dlp, change):
 
   text = clean.content_clean(revision)
 
-  # for line in text:
-  pii_results = open("pii_results.txt", "a+")
-  toxicity_results = open("toxicity_results.txt", "a+")
+  # for line in text:
```
Comment (sorensenjs) [inline antidox/wikiwatcher.py]: remove the commented out code.

File: antidox/wikiwatcher.py [hunk 3]
```diff
@@ -50,21 +49,33 @@ def log_event(apikey_data, toxicity, dlp, change):
...
+               'title:{title}'.format(**change)+", "+"comment_text:" +str(text)+", "
+               +"contains_toxicity:"+"True"+", "+ "toxic_score:"
+               str(perspective_response['attributeScores']
+                   ['TOXICITY']['summaryScore']['value'])+", ""}""\n")
+    wiki_write(result, header)
+
+def wiki_write(result, header):
+    site = pywikibot.Site()
+    repo = site.data_repository()
+    page = pywikibot.Page(site, u"User_talk:Antidox321")
```
Comment (sorensenjs) [inline antidox/wikiwatcher.py]: Pull the User_talk:Antidox321 up to the top as a constant, or maybe a flag.

Comment (sorensenjs) [reply antidox/wikiwatcher.py]: Also, how do credentials work?

**Pull request closed without merging**